dev/io.openliberty.microprofile.health.3.1.internal_fat/fat/src/io/openliberty/microprofile/health31/fat/FailedApplicationStartTest.java

```diff
@@ -0,0 +1,241 @@
+
+    @ClassRule
+    public static RepeatTests r = MicroProfileActions.repeat(FeatureReplacementAction.ALL_SERVERS,
+                                                             MicroProfileActions.MP70_EE10, // mpHealth-4.0 LITE
+                                                             MicroProfileActions.MP70_EE11, // mpHealth-4.0 FULL
+                                                             HealthActions.MP41_MPHEALTH40, //  mpHealth-4.0 FULL w/ MP41 EE8
+                                                             HealthActions.MP14_MPHEALTH40, // mpHealth-4.0 FULL w/ MP14 EE7
+                                                             MicroProfileActions.MP41, // mpHealth-3.1 FULL
+                                                             MicroProfileActions.MP40, // mpHealth-3.0 FULL
+                                                             MicroProfileActions.MP30); //mpHealth-2.0 FULL
+
+    /**
+     * Test that loads an app that throws a java.lang.Error in the servlet container intializer.
+     * We expect to see a CWWKZ0012I because of it and that the health check (overall health check) reports DOWN.
+     *
+     * This test tests the app in dropins folder.
+     *
+     * @throws Exception
+     */
+    @ExpectedFFDC("java.lang.Error")
+    @Test
+    public void webModuleStartsandFailsTest_dropins() throws Exception {
+
+        server = serverDropins;
+
+        Assert.assertTrue(String.format("Server [%s] should not have been started", server), !server.isStarted());
+
+        WebArchive app = ShrinkHelper.buildDefaultApp(APP_NAME, "io.openliberty.microprofile.health31.start.and.fails.app")
+                        .addAsServiceProvider(ServletContainerInitializer.class.getName(),
+                                              "io.openliberty.microprofile.health31.start.and.fails.app.AppServletContainerInitializer");
+
+        ShrinkHelper.exportDropinAppToServer(server, app, DeployOptions.DISABLE_VALIDATION, DeployOptions.SERVER_ONLY);
+
+        server.startServer();
+
+        log("webModuleStartsandFailsTest", "Checking (and waiting) for CWWKZ0012I");
+        String ret = server.waitForStringInLog("CWWKZ0012I");
+        assertNotNull("We expected to find CWWKZ0012I, but was not found", ret);
+
+        // Check /health -> expect that we report DOWN.
+        HttpURLConnection connhealth = HttpUtils.getHttpConnectionWithAnyResponseCode(server, HEALTH_ENDPOINT);
+        JsonObject jsonResponse = getJSONPayload(connhealth);
+        JsonArray checks = (JsonArray) jsonResponse.get("checks");
+        assertTrue("The JSON response was not empty.", checks.isEmpty());
+        assertEquals("The status of the (overall) health check was epected to be DOWN, but we received UP.", jsonResponse.getString("status"), "DOWN");
+
+        // Check /health/started -> expect that we report DOWN.
+        // Must skip for mpHealth-2.0 and mpHealth-3.1, does not contain start health check
```

Comment:
Should be mpHealth-3.0, in the comments.
```suggestion
        // Must skip for mpHealth-2.0 and mpHealth-3.0, does not contain start health check
```